examples/astro-demo/src/styles/global.css

```
@@ -0,0 +1,36 @@
+	max-width: 44rem;
+	padding: 3rem 1.5rem 8rem;
+}
+
+nav {
+	display: flex;
+	gap: 1rem;
+	margin-bottom: 2rem;
+}
+
+.panel {
+	border: 1px solid color-mix(in oklab, currentColor 20%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)